llm/llama.cpp/generate_linux.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# only do GPU builds if nvcc is installed
+if which nvcc > /dev/null; then
```

Comment:
Looking for feedback on how to get the CUDA version for the filepath in go generate, I was having trouble with this. Any alternative suggestions welcome also.